### ISSUE TITLE
Game Crashes Sometimes When Facing Bosses

### ISSUE THREAD

**Issue by DancingSnowflake497** [RussellJerome/UnrealModLoader#18]
I've been having this bad issue when fighting bosses in the game at times.  When I got so close to defeating a boss in KH3 and it occurred in Toy Box mostly, I get a lowlevelfatalerror Line 177 "Unexpected concurrency while adding new object" and the game crashes.  It happened to me two times and only once in San Fransokyo.  Also, sometimes the game hook would get stuck loading logic mods which is also another issue I have.  Sorry if my social skills aren't good.  It's my first time posting an issue.

**Comment by RussellJerome** [comment]
This doesn't seem off the bat to be a modloader specific issue as it is happening in 1 weird boss room in a random game, if you would like more help on this I recommend you join the discord and ask there.
https://discord.gg/Udy4aH5A3Q

**Comment by DancingSnowflake497** [comment]
I understand if this isn't an issue but I've tried both the update before & after the issue with the new update & it didn't work.  For some reason, vcrealtime 140_1.dll kept crashing my game with this error:  Line 177 Unexpected Concurrency while adding new object.  Once I removed unreal mod loader & the vcruntime140_1.dll file from my KH3 binaries folder, my game ran fine again.
